File: src/main/java/fi/metatavu/metaform/server/script/EncodingServices.java
package fi.metatavu.metaform.server.script;
import java.util.Base64;
import javax.enterprise.context.ApplicationScoped;

/**
 * Encoding services
 * 
 * @author Heikki Kurhinen
 */
@ApplicationScoped
public class EncodingServices {

  /**
   * Base64 encodes data
   * 
   * @param data data to encode
   */
  public String base64Encode(byte[] data) {
    return Base64.getEncoder().encodeToString(data);
  }
  
}

File: src/main/kotlin/fi/metatavu/metaform/server/script/EncodingServices.kt
package fi.metatavu.metaform.server.script

import java.util.*
import jakarta.enterprise.context.ApplicationScoped

@ApplicationScoped
class EncodingServices {
    /**
     * Base64 encodes data
     *
     * @param data data to encode
     */
    fun base64Encode(data: ByteArray): String {
        return Base64.getEncoder().encodeToString(data)
    }
}